pkgs/data/documentation/mustache/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "mustache-${version}";
+  version = "1.0.2";
+
+  outputs = [ "out" ];
```

Comment:
I can see you also have docs, so outputs should be `outputs = ["out", "man", "doc"];`